docs/about/value-proposition.md

```diff
@@ -2,50 +2,50 @@
 
 Hermez network project has some properties that will provide value to the users and the community:
 
-## Production ready solution
+## Production Ready Solution
 Hermez will launch in mainnet a project which is intended to be a usable token transfers network from day one. This product will have support, maintenance, and technological evolution to provide users with the most updated functionalities with the right risk management.
 
```

Comment:
Remove blank line

Reply:
Can you remove extra line between Production Ready solution section and Decentralized?